pages.bn/android/bugreportz.md

```diff
@@ -0,0 +1,21 @@
+# bugreportz
+
+> একটি জিপ করা অ্যান্ড্রয়েড বাগ রিপোর্ট তৈরি করুন।
+> এই কমান্ডটি শুধুমাত্র `adb shell` এর মাধ্যমে ব্যবহার করা যেতে পারে।
+> আরও তথ্য পাবেন: <https://cs.android.com/android/platform/superproject/+/master:frameworks/native/cmds/bugreportz>.
+
+- একটি অ্যান্ড্রয়েড ডিভাইসের একটি সম্পূর্ণ জিপ করা বাগ রিপোর্ট তৈরি করুন:
+
+`bugreportz`
+
+- চলমান `bugreportz` অপারেশনের অগ্রগতি দেখান:
+
+`bugreportz -p`
+
+- `bugreportz` এর সংস্করণ দেখান:
```

Comment:
Optional fix (error source: original page): `Display version:` can be used as a translation source.